libs/shared

```diff
@@ -0,0 +1 @@
+../submodules/island.is/libs/shared
```

Comment:
Curious what you need from here.

Reply:
This UI library depends on this lib, gonna try the other way and skip the symlinks and go straight to the `/submodules` directory